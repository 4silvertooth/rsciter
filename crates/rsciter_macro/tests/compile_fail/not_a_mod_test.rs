#[rsciter_macro::xmod]
struct S;

fn main() {}
