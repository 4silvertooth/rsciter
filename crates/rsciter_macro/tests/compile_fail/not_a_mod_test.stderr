error: the #[rsciter::xmod] attribute can only be applied to an inline module or impl block!
 --> tests/compile_fail/not_a_mod_test.rs:2:1
  |
2 | struct S;
  | ^^^^^^
